components/breadcrumbs/breadcrumb.js

```diff
@@ -0,0 +1,93 @@
+import '../icons/icon.js';
+import '../link/link.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class BreadCrumb extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			compact: {
+				type: Boolean,
+				reflect: true
+			},
+			href: {
+				type: String,
+				reflect: true
+			},
+			target: {
+				type: String,
+				reflect: true
+			},
+			text: {
+				type: String,
+				reflect: true
+			},
+			ariaLabel: {
+				attribute: 'aria-label',
+				type: String,
+				reflect: true
+			}
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				align-items: center;
+				display: inline-flex;
+			}
+
+			div.d2l-breadcrumb-wrapper {
+				align-items: center;
+			}
+
+			:host([compact]) div.d2l-breadcrumb-wrapper {
+				display: flex;
+				flex-direction: row-reverse;
+			}
+
+			d2l-icon {
+				height: 8px;
+				padding-left: 10px;
+				padding-right: 5px;
+				width: 8px;
+			}
+
+			d2l-icon:dir(rtl) {
+				padding-left: 5px;
+				padding-right: 10px;
+			}
+
+			:host([compact]) div.d2l-breadcrumb-wrapper d2l-icon {
+				padding-right: 10px;
+				padding-left: 0;
+			}
+
+			:host([compact]) div.d2l-breadcrumb-wrapper d2l-icon:dir(rtl) {
```

Comment:
This should also be using `:host([dir="rtl"])`, you can simply the rule to `:host([dir="rtl"][compact]) d2l-icon { ... }`

Reply:
@AllanKerr this particular one doesn't see to work on edge doing it this way. for some reason `div.d2l-breadcrumb-wrapper` should be added as well or the scoping doesn't work correctly

Reply:
The rule works without `d2l-breadcrumb-wrapper` . The class prefixing is only needed for `Slotted` rules it seems

Reply:
✔️ Done

Reply:
Yeah I think this is an awkward limitation of shadyCSS/shadyDOM which comes into play for slotted selectors + legacy Edge.